utils/lib/rr_graph/graph.py

```diff
@@ -637,23 +622,23 @@ def from_xml(cls, g, block_type_node):
         ... '''
         >>> bt = BlockType.from_xml(None, ET.fromstring(xml_string))
         >>> bt # doctest: +ELLIPSIS
-        BlockType(g=None, id=1, name='BLK_BB-VPR_PAD', size=Size(w=2, h=3), pin_classes=[...], pin_index={...})
+        BlockType(graph=None, id=1, name='BLK_BB-VPR_PAD', size=Size(w=2, h=3), pin_classes=[...], pin_index={...})
```

Comment:
Library is/was named graph, caused some namespace conflicts

Reply:
It shouldn't be a problem inside this file?